src/Standards/Squiz/Tests/Commenting/FunctionCommentUnitTest.inc

```diff
@@ -1042,7 +1052,7 @@ public function ignored() {
 
 // phpcs:set Squiz.Commenting.FunctionComment specialMethods[] __construct,__destruct
 
-/** 
```

Comment:
This space may be there on purpose to test something specific. Might be better to leave it as is.

Reply:
Well, I've run `phpunit` w/o any errors.
Could you apply a suggestion, pls, if you still think we need this space?

Thanks.